### ISSUE TITLE
unm:chrono:userlib and from-unix-ms:chrono:userlib are sometimes not invertible

### ISSUE THREAD

**Issue by tacryt-socryp** [urbit/urbit#4598]
```
(unm:chrono:userlib (from-unix-ms:chrono:userlib 1.615.573.389.789))
1.615.573.389.788
```

```
(unm:chrono:userlib (from-unix-ms:chrono:userlib 1.615.573.389.000))
1.615.573.389.000
``` 

Seems to have been introduced here, but his logic seems correct at a glance. Very confused.

https://github.com/urbit/urbit/pull/4474

@joemfb @philipcmonk @belisarius222

**Comment by tacryt-socryp** [comment]
Fixed in #4559. Hopefully it will never regress again

**Comment by matildepark** [comment]
It hasn't hit master yet — we usually wait for our issues to reflect livenet...
